homeassistant/components/homekit/type_humidifiers.py

```diff
@@ -97,6 +97,7 @@ def __init__(self, *args):
             serv_humidifier_dehumidifier.configure_char(
                 CHAR_TARGET_HUMIDIFIER_DEHUMIDIFIER,
                 value=self._hk_device_class,
+                properties={PROP_MIN_VALUE: self._hk_device_class},
```

Comment:
Should we set the max value to this as well?

Reply:
Sure, in case the default value of  `TargetHumidifierDehumidifierState` is not 0.